packages/salesforcedx-vscode-apex/package.nls.json

```diff
@@ -10,6 +10,7 @@
   "java_home_description": "Specifies the folder path to the Java 11 or Java 17 runtime used to launch the Apex Language Server (for example, /Library/Java/JavaVirtualMachines/openjdk-11.jdk/Contents/Home).",
   "java_memory_description": "Specifies the amount of memory allocation to the Apex Language Server in MB, or null to use the system default value.",
   "apex_semantic_errors_description": "Allow Apex Language Server to surface semantic errors",
+  "apex_wait_init_jobs_description": "Allow Apex Language Client not to send requests until the initializing jobs such as indexing are done.",
```

Comment:
@sbudhirajadoc Can we set signoff on this setting text? 
I personally lean towards 
`Do note allow Apex Language Client to send requests until after initialization.`

I think we have an experimental section.  I kinda think this goes under there as they user will be odd results if they turn it off.

Reply:
> I kinda think this goes under there as they user will be odd results if they turn it off.
@gbockus-sf would it make sense to add this disclaimer somewhere?

Reply:
If this is a checkbox, does checking the box stop requests from being sent to it or the other way around?  Two choices, depending on the checkbox logic. 

1. Don't allow requests to the Apex Language Server until its done initializing. OR
2. Allow requests to the Apex Language Server only until its done initializing.